### PR TITLE
docs(framework) Bump Flower dev version to 1.11.0

### PR DESCRIPTION
**Pull request by charlesbvll**
### Checklist

- [ ] Implement proposed change
- [ ] Write tests
- [ ] Update [documentation](https://flower.ai/docs/writing-documentation.html)
- [ ] Make CI checks pass
- [ ] Ping maintainers on [Slack](https://flower.ai/join-slack/) (channel `#contributions`)

### CODE REVIEW

File: baselines/doc/source/conf.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 author = "The Flower Authors"
 
 # The full version, including alpha/beta/rc tags
-release = "1.9.0"
+release = "1.11.0"
```
Comment (danieljanes) [inline baselines/doc/source/conf.py:40]:
```suggestion
release = "1.10.0"
```

File: doc/source/contributor-how-to-build-docker-images.rst [hunk 2]
```diff
@@ -65,25 +65,26 @@ Building the base image
...
+   :substitutions:
 
-  $ cd src/docker/base/<ubuntu|alpine>
-  $ docker build \
-    --build-arg PYTHON_VERSION=3.11.0 \
-    --build-arg FLWR_VERSION=1.9.0 \
-    --build-arg PIP_VERSION=23.0.1 \
-    --build-arg SETUPTOOLS_VERSION=69.0.2 \
-    -t flwr_base:0.1.0 .
+   $ cd src/docker/base/<ubuntu|alpine>
```
Comment (jafermarq) [inline doc/source/contributor-how-to-build-docker-images.rst:81]: this (and all the other) spaces are intentional?

**Review by jafermarq — COMMENTED** [review]
@charlesbvll should the title say `1.11.0` instead?

File: baselines/doc/source/conf.py (hunk 1 above)
Comment (charlesbvll) [reply baselines/doc/source/conf.py]: @danieljanes But that would make the baselines docs inconsistent with the rest, is that what we want?

File: doc/source/contributor-how-to-build-docker-images.rst (hunk 2 above)
Comment (charlesbvll) [reply doc/source/contributor-how-to-build-docker-images.rst:81]: Yes, otherwise the `code_blocks` are invalid (because of the `:substitution:` argument)

**Review by danieljanes — APPROVED** [review]

**Pull request merged**